src/tests/functional/plugin/shared/src/main.cpp

```diff
@@ -3,12 +3,30 @@
 //
 
 #include "gtest/gtest.h"
+#include "gflags/gflags.h"
 
 #include "functional_test_utils/summary/environment.hpp"
 #include "functional_test_utils/summary/op_summary.hpp"
 #include "functional_test_utils/skip_tests_config.hpp"
 
+DEFINE_int32(gpu_device_id, -1, "GPU Device ID (a number starts from 0)");
```

Comment:
Since it's a common tests binary, I think the option should be named w/o gpu prefix, like `device_id`. Also, data type should be string as "DEVICE_ID" property expects string, and other plugins (e.g. Myriad) may use another naming conventions.

Reply:
Done.